units/event_send.go

```diff
@@ -49,11 +49,11 @@ func makeEventNotificationJob() *eventNotificationJob {
 	return j
 }
 
-func NewEventNotificationJob(id string) amboy.Job {
+func NewEventNotificationJob(id, ts string) amboy.Job {
```

Comment:
the second argument should be a time.Time rather than a string.

Reply:
it seems most jobs use a string derived from `.Format(TSFormat)`